app/src/main/kotlin/com/wire/android/ui/theme/WireColorScheme.kt

```diff
@@ -8,61 +8,63 @@ import io.github.esentsov.PackagePrivate
+    val surface: Color, val onSurface: Color,
+    val primaryButtonEnabled: Color, val onPrimaryButtonEnabled: Color,
+    val primaryButtonDisabled: Color, val onPrimaryButtonDisabled: Color,
+    val primaryButtonSelected: Color, val onPrimaryButtonSelected: Color,
     val primaryButtonRipple: Color,
-    val secondaryButtonEnabled: Color,                  val onSecondaryButtonEnabled: Color,
+    val secondaryButtonEnabled: Color, val onSecondaryButtonEnabled: Color,
     val secondaryButtonEnabledOutline: Color,
-    val secondaryButtonDisabled: Color,                 val onSecondaryButtonDisabled: Color,
+    val secondaryButtonDisabled: Color, val onSecondaryButtonDisabled: Color,
     val secondaryButtonDisabledOutline: Color,
-    val secondaryButtonSelected: Color,                 val onSecondaryButtonSelected: Color,
+    val secondaryButtonSelected: Color, val onSecondaryButtonSelected: Color,
     val secondaryButtonSelectedOutline: Color,
     val secondaryButtonRipple: Color,
-    val tertiaryButtonEnabled: Color,                   val onTertiaryButtonEnabled: Color,
-    val tertiaryButtonDisabled: Color,                  val onTertiaryButtonDisabled: Color,
-    val tertiaryButtonSelected: Color,                  val onTertiaryButtonSelected: Color,
+    val tertiaryButtonEnabled: Color, val onTertiaryButtonEnabled: Color,
+    val tertiaryButtonDisabled: Color, val onTertiaryButtonDisabled: Color,
+    val tertiaryButtonSelected: Color, val onTertiaryButtonSelected: Color,
     val tertiaryButtonSelectedOutline: Color,
     val tertiaryButtonRipple: Color,
     val divider: Color,
     val secondaryText: Color,
     val labelText: Color,
-    val badge: Color,                                   val onBadge: Color,
+    val badge: Color, val onBadge: Color,
     val highLight : Color,
     val checkedCheckBoxBorderColor : Color,
     val uncheckedCheckBoxBorderColor : Color,
     val disabledIndeterminateCheckBoxBorderColor : Color,
-    val disabledCheckBoxBorderColor : Color,
-    val checkedBoxColor : Color,
-    val uncheckedBoxColor : Color,
-    val disabledCheckedBoxColor : Color,
-    val disabledIndeterminateBoxColor : Color,
-    val disabledUncheckedBoxColor : Color,
-    val uncheckedCheckmarkColor : Color,
-    val checkedCheckmarkColor : Color,
-    val ongoingCallBackground : Color,
-    val initiatingCallBackground : Color,
-    val callingIncomingBackground : Color
+    val disabledCheckBoxBorderColor: Color,
+    val checkedBoxColor: Color,
+    val uncheckedBoxColor: Color,
+    val disabledCheckedBoxColor: Color,
+    val disabledIndeterminateBoxColor: Color,
+    val disabledUncheckedBoxColor: Color,
+    val uncheckedCheckmarkColor: Color,
+    val checkedCheckmarkColor: Color,
+    val ongoingCallBackground: Color,
+    val initiatingCallBackground: Color,
+    val callingIncomingBackground: Color,
+    val deletedMessageTextBorder: Color,
+    val deletedMessageTextColor: Color,
```

Comment:
We shouldn't add any new colors to the scheme unless it's added to the design documentation. The existing scheme should already cover all required colors so we should just use the existing ones and not duplicate them. For instance, I believe instead of creating new`deletedMessageTextColor`, `labelText` should just be used unless the design documentation clearly states that they should be separate semantic colors.

Reply:
Thanks, for the `deleteMessageTextColor` id used `labelText` but for delete `border` I couldn't find any good candidates, I used `secondaryButtonDisabled` color, please let me know if you know any better candidate or I should create a new one?